cmd/blockchaincmd/change_weight.go

```diff
@@ -143,7 +143,16 @@ func setWeight(_ *cobra.Command, args []string) error {
 		}
 	}
 
-	validationID, err := validator.GetValidationID(rpcURL, nodeID)
+	if sc.ValidatorManagerAddress == "" {
```

Comment:
not sure about this, probably the command should fail if the validator manager address is empty